core/src/main/java/com/graphhopper/routing/util/countryrules/CountryRuleFactory.java

```diff
@@ -25,14 +25,64 @@
 import java.util.Map;
 
 import com.graphhopper.routing.ev.Country;
+import com.graphhopper.routing.util.countryrules.europe.*;
 
 public class CountryRuleFactory {
     
     private final Map<Country, CountryRule> rules = new EnumMap<>(Country.class);
     
     public CountryRuleFactory() {
+        
+        // Europe
+        rules.put(ALB, new AlbaniaCountryRule());
+        rules.put(AND, new AndorraCountryRule());
         rules.put(AUT, new AustriaCountryRule());
+        rules.put(BEL, new BelgiumCountryRule());
+        rules.put(BGR, new BulgariaCountryRule());
+        // BIH
+        rules.put(BLR, new BelarusCountryRule());
+        rules.put(CHE, new SwitzerlandCountryRule());
+        rules.put(CZE, new CzechiaCountryRule());
         rules.put(DEU, new GermanyCountryRule());
+        rules.put(DNK, new DenmarkCountryRule());
+        rules.put(ESP, new SpainCountryRule());
+        rules.put(EST, new EstoniaCountryRule());
+        rules.put(FIN, new FinlandCountryRule());
+        rules.put(FRA, new FranceCountryRule());
+        rules.put(FRO, new FaroeIslandsCountryRule());
+        rules.put(GGY, new GuernseyCountryRule());
+        rules.put(GIB, new GibraltarCountryRule());
+        // GBR
+        rules.put(GRC, new GreeceCountryRule());
+        rules.put(HRV, new CroatiaCountryRule());
+        rules.put(HUN, new HungaryCountryRule());
+        rules.put(IMN, new IsleOfManCountryRule());
+        // IRL
+        rules.put(ISL, new IcelandCountryRule());
+        rules.put(ITA, new ItalyCountryRule());
+        rules.put(JEY, new JerseyCountryRule());
+        rules.put(LIE, new LiechtensteinCountryRule());
+        rules.put(LTU, new LithuaniaCountryRule());
+        rules.put(LUX, new LuxembourgCountryRule());
+        rules.put(LVA, new LatviaCountryRule());
+        rules.put(MCO, new MonacoCountryRule());
+        // MDA
+        // MKD
```

Comment:
I guess these are just missing countries for which we have no toll rules yet? Can you just add an empty country rule for these? Or doesn't this make sense?

Reply:
I can add empty CountryRules for them. Probably easier to fill the gaps for potential contributors.